src/plugins/opensearch_dashboards_react/public/exit_full_screen_button/exit_full_screen_button.tsx

```diff
@@ -57,6 +57,7 @@ class ExitFullScreenButtonUi extends PureComponent<ExitFullScreenButtonProps> {
   }
 
   public render() {
+    const colorScheme = this.props.logos.colorScheme !== 'dark' ? 'dark' : 'light';
```

Comment:
It seems like there should be a better way to do this, but I understand that the button color itself is currently styled directly.

Reply:
all of these would eventually move to pure CSS.